challenges/objects-arrays.js

```diff
@@ -97,29 +164,31 @@ The zoos need a list of all their animal's names (names only, not scientific) co
 
 */
 
-const lowerCase = [];
+let lowerCase = zooAnimals.map((animal) => {
+  return animal.animal_name.toLowerCase();
+});
 console.log(lowerCase); 
 
 /* Request 3: .filter() 
 
 The zoos are concenred about animals with a lower population count. Find out which animals have a population less than 5.
 
 */
-const lowerPopulation = [];
+
+const lowerPopulation = zooAnimals.filter(lowerPopulation => lowerPopulation.zooanimals < 5);
+
 console.log(lowerPopulation);
 
 /* Request 4: .reduce() 
 
 The zoos need to know their total animal population across the United States.  Find the total population from all the zoos using the .reduce() method.
 
 */
-const populationTotal = 0;
-console.log(populationTotal);
-
+const zoos = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9];
+const reducer = (acc, curr) => acc + curr
 
+console.log(zoos.reduce(reducer, 0));
```

Comment:
Using MDN as the syntax example is great! But you need to adapt the syntax to your problem. In this scenario, you want something like this:

```suggestion
const populationTotal = []; // Empty array for easy adding and reducing later on
zooAnimals.forEach(element => populationTotal.push(element.population)); // Loop through root array using forEach, pushing population values into new array
console.log(populationTotal.reduce(reducer, 0)); // Log the total population count for all zoos
```